ui/apps/platform/src/Containers/NetworkGraph/simulation/DeploymentScopeModal.tsx

```diff
@@ -38,26 +85,64 @@ function DeploymentScopeModal({ deployments, isOpen, onClose }: DeploymentScopeM
                 </Button>,
             ]}
         >
-            <TableComposable variant="compact">
-                <Thead noWrap>
-                    <Tr>
-                        <Th width={50} sort={getSortParams('name')}>
-                            Deployment
-                        </Th>
-                        <Th width={50} sort={getSortParams('namespace')}>
-                            Namespace
-                        </Th>
-                    </Tr>
-                </Thead>
-                <Tbody>
-                    {sortedDeployments.map(({ name, namespace }) => (
-                        <Tr key={`${namespace}/${name}`}>
-                            <Td dataLabel="Deployment">{name}</Td>
-                            <Td dataLabel="Namespace">{namespace}</Td>
+            <Toolbar>
+                <ToolbarContent>
+                    <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
+                        <Pagination
+                            isCompact
+                            itemCount={scopeDeploymentCount}
+                            page={page}
+                            perPage={perPage}
+                            onSetPage={(_, newPage) => setPage(newPage)}
+                            onPerPageSelect={(_, newPerPage) => {
+                                if (scopeDeploymentCount < (page - 1) * newPerPage) {
+                                    setPage(1);
+                                }
+                                setPerPage(newPerPage);
+                            }}
+                        />
+                    </ToolbarItem>
+                </ToolbarContent>
+            </Toolbar>
+            {error && (
+                <Bullseye>
+                    <EmptyStateTemplate
+                        title="There was an error loading deployments"
+                        headingLevel="h2"
+                        icon={ExclamationCircleIcon}
+                        iconClassName="pf-u-danger-color-100"
+                    >
+                        {getAxiosErrorMessage(error.message)}
```

Comment:
The initial iteration of this util function is for Axios and I did not dig very deep how much different for apollo

Although the argument has `unknown` type, `error` is expected instead of `error.message`

Do you have enough experience with GraphQL query errors for us to collaborate on a pattern to follow?

Reply:
Ah good catch, according to the Apollo docs if `error` is truthy it will be of a type that `extends Error` and contains a `message` property. I'll remove the axios wrapper here.